src/org/thoughtcrime/securesms/contacts/ContactAccessor.java

```diff
@@ -66,13 +69,30 @@ public String getAllSystemContactsAsString(Context context) {
     Set<String> contactPhotoIdentifiers = new HashSet<>();
     while (systemContactsCursor != null && systemContactsCursor.moveToNext()) {
 
-      String name = systemContactsCursor.getString(CONTACT_CURSOR_NAME);
-      name = name.replace("\r", ""); // remove characters later used as field separator
-      name = name.replace("\n", "");
+      String name;
+      try {
+        name = systemContactsCursor.getString(CONTACT_CURSOR_NAME);
+        if (name != null) {
+          name = name.replace("\r", ""); // remove characters later used as field separator
+          name = name.replace("\n", "");
+        } else {
+          name = "";
+        }
+      } catch(Exception e) {
```

Comment:
The null-check would be enough for the special stack trace we got but in the docs for Cursor.getString() it said

"The result and whether this method throws an exception when the column value is null or the column type is not a string type is implementation-defined."